libs/table/src/lib/directives/tree-grid/tree-grid.cell.directive.ts

```diff
@@ -0,0 +1,179 @@
+import { Directive, ElementRef, HostBinding, HostListener, Input, Optional } from '@angular/core';
+
+import { NgxHasFocusDirective } from '../has-focus-action';
+import { NgxTreeGridCellTarget, NgxTreeGridRowTarget } from '../../interfaces';
+import { NgxTreeGridRowDirective } from './tree-grid-row.directive';
+import { NgxTreeGridDirective } from './tree-grid.directive';
+
+/**
+ * A cell directive to handle navigation according to the WCAG treegrid pattern
+ *
+ * See https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/
+ */
+@Directive({
+	selector: '[ngxTreeGridCell]',
+	standalone: true,
+})
+export class NgxTreeGridCellDirective extends NgxHasFocusDirective {
+	/**
+	 * The parent row of the cell
+	 */
+	private row: NgxTreeGridRowDirective;
+
+	/**
+	 * Marks the cell as focusable, but only by setting its focus programmatically, not by the tab key
+	 */
+	@HostBinding('attr.tabIndex') private canFocus: number = -1;
```

Comment:
This can also be moved to the `host` property of the decorator. A matter of preference, but since it's a static value, might be something to consider.